### ISSUE TITLE
NVDA does not recognize relationships within matrices in PowerPoint

### ISSUE THREAD

**Issue by Tiff-Koval** [nvaccess/nvda#19108]
### Brief summary

When navigating through a matrix in Microsoft PowerPoint the row/columns are not announced with NVDA. 

### Steps to reproduce

1. Open up Microsoft PowerPoint
2. Navigate to Insert in the top menu
3. Activate Equation and navigate to Matrix
4. Create an Identity Matrix
5. Navigate through the matrix with NVDA.
6. Notice after each item, NVDA announces the entire matrix contents rather than where the user is in the matrix

[Identity Matrix.pptx](https://github.com/user-attachments/files/22933587/Identity.Matrix.pptx)

### Actual behavior

When navigating through a matrix, the row/columns are not announced with NVDA. Instead NVDA announces the entire contents of the matrix without context.

After finishing the first cell in the matrix, NVDA announces the entire matrix, such as, "1 0 0 0 1 0 0 0 1"

### Expected behavior

When navigating through a cell in the matrix, NVDA should announce where they are in the matrix, such as "End element 1 1". 
Expected behavior is seen with JAWS + PowerPoint and NVDA + Microsoft Word

[Word Matrix.docx](https://github.com/user-attachments/files/22933703/Word.Matrix.docx)

### NVDA type

installed copy

### NVDA version

2025.3

### Windows version

10.0.26100 Build 26100

### Name and version of other software in use when reproducing the issue

PowerPoint Version 2507

### Other information about your system

Other troubvleshooting:

JAWS + PowerPoint work as expected
NVDA + Microsoft Word work as expected

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by josephsl** [comment]
Hi,

Thanks for submitting the issue. Have you tested this issue while using Narrator + PowerPoint as well?

Thanks.

**Comment by Emil-18** [comment]
@Tiff-Koval If it works with narrator, you can install the enhanced control support add-on, (not available on the add-on store yet, but avaliable on the spannish NVDA add-ons website). Navigate to the objects that doesn't work, and press NVDA+alt+c. select "UIA" in the combo box, and press enter

**Comment by Tiff-Koval** [comment]
@Emil-18 @josephsl thank you both. The matrix does read as expected with Narrator and with NVDA + the enhanced control add on. It sounds like we somewhat have a workaround. Hoping NVDA can fix without needing an add on?
